.github/workflows/periodic_benchmarks.yml

```diff
@@ -27,12 +27,14 @@ jobs:
       - name: Install Linux system dependencies
         run: |
           sudo apt-get update
-          sudo apt install gfortran gcc libopenblas-dev
+          sudo apt-get install gfortran gcc libopenblas-dev
       - name: Install python dependencies
         run: |
           python -m pip install --upgrade pip wheel setuptools virtualenv asv wget cmake casadi numpy
       - name: Install SuiteSparse and Sundials
         run: python scripts/install_KLU_Sundials.py
+      - name: Install tox and asv
+        run: pip install -U pip "tox<4" asv
```

Comment:
I think this should not be here? We are installing `asv` in one of the steps above and I don't think this workflow uses `tox`.

Reply:
Apologies, I don't know where and when I added this, or how, even. I have been working on this PR for too long haha